core/templates/dev/head/base.html

```diff
@@ -122,7 +122,7 @@
     {% if iframed %}
       {{ warnings_and_loader() }}
     {% else %}
-      <div class="oppia-base-container" ng-class="{'oppia-sidebar-menu-open': sidebarIsShown, 'oppia-sidebar-menu-closed': !sidebarIsShown}">
+      <div class="oppia-base-container" ng-class="{'oppia-sidebar-menu-open': sidebarIsShown, 'oppia-sidebar-menu-closed': !sidebarIsShown}" ng-swipe-left="openSidebar()" ng-swipe-right="openSidebar()">
```

Comment:
Hi @Anurabh98, some questions:

(1) How did you test this?
(2) Should the sidebar close on swiping right?
(3) What happens if the screen is wide enough that there's no sidebar shown?